src/components/program-settings.js

```diff
@@ -67,8 +67,8 @@ class ProgramSettings extends React.Component {
         teReservedDBTrimming: teReservedDBTrimming,
         eventsDownload: eventsDownload,
         eventsDBTrimming: eventsDBTrimming,
-        eventPeriodDownload: eventPeriodDownload,
-        eventPeriodDBTrimming: eventPeriodDBTrimming,
+        eventDateDownload: eventDateDownload,
```

Comment:
This works just fine since it's used throughout this file, but for future reference it is possible to shorten this a bit:

```js
state = {
   eventDateDownload: eventDateDownload,
}
```

is the same as 

```js
state = {
    eventDateDownload,
}
```

And that way you don't need to repeat the name twice.

Reply:
okey dokey! I'll keep it in mind for next PRs